### ISSUE TITLE
Error in Corollary 7.74 

### ISSUE THREAD

**Issue by farmanb** [dcernst/IBL-IntroToProof#149]
Unless I am mistaken -- Spring Break starts in 2 days, so it's entirely possible my brain just isn't functioning -- I believe there is an error in Corollary 7.74 on p.105 of the printed version.  As printed, it states

> If $R$ is a relation on a nonempty set $A$ such that the collection of the set of relatives with respect to $R$ is a partition of $A$, then $R$ is an equivalence relation.

Under these hypotheses, it's certainly true that if $\sim$ is the relation on $A$ associated to $\operatorname{Rel}(R)$ (as in Definition 7.62), then $\sim$ is an equivalence relation by Theorems 7.68, 7.70, and and 7.71.  However, it's not necessarily true that $R$ was an equivalence relation (i.e. $R$ and $\sim$ may not be the same).

As a counter-example, take $A =\ {a, b\}$.  The relation $R = ${(a, b), (b,a)}$ is symmetric, and neither reflexive nor transitive.  The sets of relatives are the singletons $\operatorname{rel}(a) = \{b\}$ and $\operatorname{rel}(b) = \{a\}$, and  the collection of the set of relatives with respect to $R$ is $\operatorname{Rel}(R) = \{ \{a\}, \{b\} \}$.  Following Definition 7.51, this is a partition of $A$ because 

a) $\operatorname{rel}(a) = \{b\}$ and $\operatorname{rel}(b) = \{a\}$ are both non-empty,
b) $\operatorname{rel}(a) = \{b\}$ and $\operatorname{rel}(b) = \{a\}$ are disjoint, and
c) $\operatorname{rel}(a) = \{b\}$ and $\operatorname{rel}(b) = \{a\}$ cover $A$.

Following Definition 7.62, the associated relation, $\sim$, becomes an equivalence relation, but swaps the sets of relatives: $a \sim a$ because $a$ is an element of $\operatorname{rel}(b)$ and $b \sim b$ because $b$ is an element of $\operatorname{rel}(a)$.  That is, the associated relation is the diagonal $\{(a, a), (b, b)\} \subseteq A \times A$, which is not the same as $R$.

-Blake

Edit: Didn't realize GH supported tex.

**Comment by dcernst** [comment]
Hmmm, I agree this is broken.  I'm looking through previous versions to see when I introduced this in an attempt to figure out what it is supposed to say or why this corollary is even there. I'm not sure what the right fix is. It seems like removing this result does no damage. Any thoughts?

**Comment by dcernst** [comment]
Maybe an interesting punchline would be "...then, $R_{Rel(R)}$ is an equivalence relation."

**Comment by farmanb** [comment]
I don't know the original intent, but when I first glanced at it, I interpreted the statement as a sort of converse to Theorem 7.59:

> If $\sim$ is an equivalence relation on a nonempty set $A$, then $A/\sim$ forms a partition of $A$.

I.e. something along the lines of "If $\Omega$ is a partition of $A$, then $R_\Omega$ is an equivalence relation on $A$."  The second thought in my mind was there would be a follow-up exercise in/around 8.2 that asked students to rephrase this equivalence as a bijection between the set of equivalence relations on $A$ and the set of partitions of $A$.

I think that  "... then, $R_{\operatorname{Rel}(R)}$ is an equivalence relation" would also be a good to include, to drive home the point that the bijection fails if we expand the equivalence relations to include all relations on $A$ since both $R$ and $R_{\operatorname{Rel}(R)}$ generate the same partition.

**Comment by farmanb** [comment]
I have committed the cardinal sin of trying to math before being sufficiently caffeinated.  I just realized that I pitched duplicating Theorem 7.73 as a corollary to itself...
> I don't know the original intent, but when I first glanced at it, I interpreted the statement as a sort of converse to Theorem 7.59:
> 
> > If ∼ is an equivalence relation on a nonempty set A, then A/∼ forms a partition of A.
> 
> ...

Either way, I think modifying it to "... then $R_{\operatorname{Rel}(R)}$ is an equivalence relation" would be a good idea for the reasons above.  Additionally, I think it might be a good exercise to ask students to either come up with a counterexample to the current statement of Corollary 7.74 or to verify that a given relation $R$ relation is not an equivalence relation and partitions $A$.

**Comment by dcernst** [comment]
I think I figured out what this result should have been.  Here's a summary of what I just added to errata:

- Page 105, Corollary~7.74: As stated the corollary is incorrect. However, including the hypothesis ``$a\in\rel(a)$ for all $a\in A$" makes the statement true. However, I think it is incorrect to refer to this result as a ``Corollary", so it has been renamed ``Theorem". As a result, the reference to the result in Example~7.80 and the paragraph prior to Theorem~7.81 have been updated.
- Page 106, Example~7.80 and paragraph prior to Theorem~7.81: ``Corollary~7.74" replaced with ``Theorem~7.74."

Source updated. New PDF and errata added to book page:

http://danaernst.com/IBL-IntroToProof/

Thanks for bringing this error to my attention!
